rust/processor/src/processors/fungible_asset_processor.rs

```diff
@@ -452,11 +500,11 @@ impl ProcessorTrait for FungibleAssetProcessor {
 pub async fn parse_v2_coin(
```

Comment:
nit: once we delete the old processors, this function can be moved to `common` models folder

Reply:
added a TODO